src/components/TimeSelectInput/readme.md

```diff
@@ -0,0 +1,57 @@
+##### TimeSelectInput base:
+
+```js
+import React from 'react';
+import { TimeSelectInput } from 'react-rainbow-components';
+import styled from 'styled-components';
+
+const StyledContainer = styled.div`
+    display: flex;
+    justify-content: center;
+`;
+<StyledContainer>
+    <TimeSelectInput
+        id="time-select-input-1"
+    />;
+</StyledContainer>
```

Comment:
All Inputs use the "value" and "onChange ()" pattern. To do the examples it had to be like this.

`const TimeSelectInputExample = () => {
    const [time, setTime] = useState('08:00 AM');
    return (
        <TimeSelectInput
            id="time-select-input-2"
            value="08:00 AM"
            onChange={setTime}
        />
    );
}
<StyledContainer>
    <TimeSelectInputExample />
</StyledContainer>
`